srml/support/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 
 [dependencies]
+log = "0.4"
```

Comment:
Don't we need to disable the `std` feature?

Reply:
I think it's actually off by default: https://github.com/rust-lang-nursery/log/blob/master/Cargo.toml#L40